distribution/lib/Standard/Database/0.0.0-dev/src/Connection/PostgreSQL.enso

```diff
@@ -0,0 +1,81 @@
+from Standard.Base import all
+
+import Standard.Database.Data.Dialect
+import Standard.Database.Connection.Connection
+from Standard.Database.Connection.Credentials as Credentials_Module import Credentials
+import Standard.Database.Connection.Connection_Options
+import Standard.Database.Connection.SSL_Mode
+from Standard.Database.Connection.SSL_Mode import all
+import Standard.Database.Connection.Client_Certificate
+
+polyglot java import org.postgresql.Driver
+
+type PostgreSQL
+    ## Connect to a PostgreSQL database.
+
+       Arguments:
+       - host: The hostname of the database server (defaults to localhost).
+       - port: The port of the database server (defaults to 5432).
+       - database: The database to connect to (if not provided or empty, the default database will be used).
```

Comment:
It is unclear to me what 'not provided' means here? Does it mean that `Nothing` is provided (seems not as the type nor code do not allow it), or does it mean that simply the default is not overridden by providing the argument? I guess the latter but the wording may be unclear. Maybe just reword it to say what is the default used like in previous arguments? Before, for example in the case of the `port` we don't say 'if not provided' but simply specify the default and this seems more consistent with our other docs and more clear in meaning.